framework/sendemail.py

```diff
@@ -62,19 +62,26 @@ def handle_outbound_mail_task():
   if settings.SEND_ALL_EMAIL_TO and to != settings.REVIEW_COMMENT_MAILING_LIST:
     to_user, to_domain = to.split('@')
     to = settings.SEND_ALL_EMAIL_TO % {'user': to_user, 'domain': to_domain}
-    # Don't add CC addresses for non-prod environments.
-    cc = None
+    if cc:
+      new_cc = []
+      for cc_addr in cc:
+        cc_user, cc_domain = cc_addr.split('@')
+        new_cc.append(
+            settings.CC_ALL_EMAIL_TO % {'user': cc_user, 'domain': cc_domain})
+      cc = new_cc
 
   sender = 'Chromestatus <admin@%s.appspotmail.com>' % settings.APP_ID
   if from_user:
     sender = '%s via Chromestatus <admin+%s@%s.appspotmail.com>' % (
         from_user, from_user, settings.APP_ID)
 
   message = mail.EmailMessage(
-      sender=sender, to=to, cc=cc, subject=subject, html=email_html)
+      sender=sender, to=to, subject=subject, html=email_html)
   if reply_to:
     message.reply_to = reply_to
   message.check_initialized()
+  if cc:
+    message.cc = cc
```

Comment:
Just to confirm, message.cc can be both cc from the param and cc after the processing? Because it is being reassigned on line 71

Reply:
The reassignment on line 71 should only occur in staging environments, so this should be the exact input given for cc on prod.